Makefile

```diff
@@ -175,6 +175,13 @@ docker-buildx: ## Build and push docker image for the manager for cross-platform
 	- $(CONTAINER_TOOL) buildx build --push --platform=$(PLATFORMS) --tag ${OPERATOR_IMAGE_TAG} -f Dockerfile.cross .
 	- $(CONTAINER_TOOL) buildx rm project-v3-builder
 	rm Dockerfile.cross
+	# same for config-daemon
+	sed -e '1 s/\(^FROM\)/FROM --platform=\$$\{BUILDPLATFORM\}/; t' -e ' 1,// s//FROM --platform=\$$\{BUILDPLATFORM\}/' Dockerfile.nic-configuration-daemon > Dockerfile.nic-configuration-daemon.cross
```

Comment:
Because we don't necessarily want to build cross-platform (make docker-build target). Essentially, it's the code, auto-generated by operator-sdk, slightly changed to also build the daemon